src/handler.ts

```diff
@@ -140,15 +140,15 @@ const FsWatchInstances = new Map<string, FsWatchContainer>();
  * @param listener main event handler
  * @param errHandler emits info about errors
  * @param emitRaw emits raw event data
- * @returns new fsevents instance
+ * @returns {NativeFsWatcher}
```

Comment:
can we remove all `@returns`?

Reply:
we could

I'll see if I can remove them all in a separate PR 👍